demos/llm_npu/README.md

```diff
@@ -11,7 +11,7 @@ It is targeted on client machines equipped with NPU accelerator.
 
 ## Prerequisites
 
-**Model preparation**: Python 3.12 or higher with pip and HuggingFace account
+**Model preparation**: HuggingFace account
```

Comment:
Actually, are you sure that we even need HF account for this? From what I can see we are using model from OpenVINO org and they do not require token to access, so maybe even that is unnecessary?